### ISSUE TITLE
Key error 'file'

### ISSUE THREAD

**Issue by RicherMans** [TUT-ARG/sed_eval#9]
Hey there,
I just noticed an error, that during evaluation for scene detection, the following error happens:

```
Traceback (most recent call last):
  File "../../../../system/evaluate17.py", line 36, in <module>
    file_pair['estimated_scene_list'])
  File ".local/lib/python3.6/site-packages/sed_eval/scene.py", line 159, in evaluate
    if estimated_item['file'] == reference_item['file']:
KeyError: 'file'
```
I checked out the source for `.load()` in dcase_util and it seems that all variables have been renamed to `filename` not file. A simple fix would be to just replace the name.

Or did I misunderstand some of the usage of this script?

**Comment by windclay** [comment]
I changed 'file' to 'filename', and then it works well

**Comment by toni-heittola** [comment]
This issue has been addressed with commit 31592712a091001a87293947a27c7ae8d874047f. 

Originally field was `file`. However,  in `dcase_util` toolbox used to handle meta data the same field is named `filename`. Current version of `dcase_util` mirrors `filename` field to `file` for backward compatibility.
